main.js

```diff
@@ -2,12 +2,9 @@
 // main.js is basicly what makes the bot run. There are more comments throughout that help descripe some elements.
 //
 const Discord = require('discord.js');
-
-const mongoose = require('mongoose'); 
-const fs = require('fs');
-
 const config = require('dotenv').config();
-const { prefix, token, roles, MongoDB } = require('./config.json');
+const { token } = require('./config.json');
```

Comment:
Shouldn't this still be `const { prefix, token, roles, MongoDB } = require('./config.json');`?

Reply:
Made the two changes in main.js